addons/mail/static/src/core/common/im_status_service.js

```diff
@@ -42,24 +42,26 @@ export const imStatusService = {
             "bus.bus/im_status_updated",
             async ({ presence_status, im_status, partner_id, guest_id, debounce = true }) => {
                 const store = env.services["mail.store"];
-                const persona = store.Persona.get({
-                    type: partner_id ? "partner" : "guest",
-                    id: partner_id || guest_id,
-                });
-                if (!persona) {
-                    return; // Do not store unknown persona's status
+                const partner = store["res.partner"].get({ id: partner_id });
+                const guest = store["mail.guest"].get({ id: guest_id });
```

Comment:
I don't think we still need an object with id for get if the models become single-id models.